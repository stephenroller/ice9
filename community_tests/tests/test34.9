# assign then write all 3 basic types

var a, b : int
var c, d : bool
var e, f : str

a := 0;
b := 1;
c := true;
d := false;
e := "Hi";
f := "There";

write a;
write b;
write ? c;
write ? d;
write e;
write f;
