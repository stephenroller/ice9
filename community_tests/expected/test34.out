Loading file: generated/test34.tm
0 
1 
1 
0 
Hi
There
Number of instructions executed = 122
